tensorboard/plugins/graph/tf_graph_board/tf-graph-board.ts

```diff
@@ -16,6 +16,7 @@ import {PolymerElement, html} from '@polymer/polymer';
 import {customElement, observe, property} from '@polymer/decorators';
 
 import '../../../components/polymer/irons_and_papers';
+
```

Comment:
Remove this extra line. It does not seem to be relevant to the internal import grouping nor VSCode's default import grouping.